build/pipelines/release.yml

```diff
@@ -100,36 +100,7 @@ jobs:
         If ($Arch -Eq "x86") { $Arch = "Win32" }
 
         Write-Host "##vso[task.setvariable variable=RationalizedBuildPlatform]${Arch}"
-  - task: NuGetToolInstaller@1
-    displayName: Use NuGet 5.10
-    inputs:
-      versionSpec: 5.10
-  - task: NuGetAuthenticate@0
-  # In the Microsoft Azure DevOps tenant, NuGetCommand is ambiguous.
-  # This should be `task: NuGetCommand@2`
-  - task: 333b11bd-d341-40d9-afcf-b32d5ce6f23b@2
-    displayName: Restore NuGet packages for extraneous build actions
-    inputs:
-      command: restore
-      feedsToUse: config
-      configPath: NuGet.config
-      restoreSolution: build/packages.config
-      restoreDirectory: '$(Build.SourcesDirectory)\packages'
-  - task: NuGetCommand@2
-    displayName: NuGet custom
-    inputs:
-      command: custom
-      selectOrConfig: config
-      nugetConfigPath: NuGet.Config
-      arguments: restore OpenConsole.sln -SolutionDirectory $(Build.SourcesDirectory)

```

Comment:
This one (and the one that was copied and pasted into `build-console-compliance-job.yml`) look a little different to the others, but I think they have the same end result.